signer/src/storage/mod.rs

```diff
@@ -232,7 +232,7 @@ pub trait DbRead {
         &self,
     ) -> impl Future<Output = Result<Option<model::EncryptedDkgShares>, Error>> + Send;
 
-    /// Returns the number of DKG shares entries in the database.
+    /// Returns the number of non failed DKG shares entries in the database.
```

Comment:
```suggestion
    /// Returns the number of non-failed DKG shares entries in the database.
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/1317/commits/660698878265dffdc7768171691efb4bee38f040